frontend/packages/core/src/AppLayout/logo.tsx

```diff
@@ -1,5 +1,6 @@
 import React from "react";
-import styled, { keyframes } from "styled-components";
+import { keyframes } from "@emotion/core";
```

Comment:
the rotate animation didn't work when I imported keyframes from `'@emotion/react'`. came across this https://stackoverflow.com/questions/55119780/react-emotion-animating-a-div-with-emotionjs which solved the issue.

Reply:
can you import this from `@emotion/react` instead since we already depend on that? Looking at the [emotion docs](https://emotion.sh/docs/keyframes) it appears to be the recommendation

Reply:
oh it works now. i probs had other changes at the time that broke the animation when i first imported it from emotion/react.

updated and removed the dep from package.json.